samples/matter/light_switch/Kconfig

```diff
@@ -23,6 +23,23 @@ config STATE_LEDS
 	  the device into a network or the factory reset initiation. Note that setting this option to
 	  'n' does not disable the LED indicating the state of the simulated bolt.
 
+if MPSL_FEM
+
+config FEM_802_15_4_DEFAULT_TX_POWER
```

Comment:
Do we need to copy+paste this configuration for every sample? Maybe we could add this to the Kconfig.features in the sdk-connectedhomeip?